packages/point-of-sale/src/graphql/queries/getWalletAddress.ts

```diff
@@ -0,0 +1,53 @@
+import { gql } from '@apollo/client'
+
+export const GET_WALLET_ADDRESS_BY_URL = gql`
+  query GetWalletAddress(
+    $url: String!
+    $first: Int
+    $last: Int
+    $before: String
+    $after: String
+    $sortOrder: SortOrder
+  ) {
+    walletAddressByUrl(url: $url) {
+      id
+      incomingPayments(
+        first: $first
+        last: $last
+        before: $before
+        after: $after
+        sortOrder: $sortOrder
+      ) {
+        edges {
+          node {
+            id
+            url
+            walletAddressId
```

Comment:
let's also fetch the senderWalletAddress now that @oana-lolea's PR #3705 has gone in